systemd/ics2entropiawiki.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Retrieve ics File and publish it to the entropia wiki
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/ics2entropiawiki -c /etc/ics2entropiawiki/config.ini
+User=ics2entropiawiki
```

Comment:
document the requirement for this user to be present in the README.
Ideally, with a copy-pastable command to add a system user called `ics2entropiawiki`.

Reply:
I would do that, ideally I would provide something script like doing everything that needs to be done. A `.deb` maybe?